docs/0-what-is/1-getting-started.mdx

```diff
@@ -5,81 +5,191 @@ sidebar_position: 0
 slug: '/getting-started'
 ---
 
-Pipeline configuration files give you the ability to define pipelines that are
-provisioned by Conduit at startup. It's as simple as creating a YAML file that
-defines pipelines, connectors, processors, and their corresponding
-configurations.
+## Our goal
 
-## Example pipeline
+In this guide, our goal will be to stream information about flights from an
+imaginary airport to a file. The flight records contain just the airline name
+and the scheduled departure time.
 
-:::tip
+## Install Conduit
 
-In our [Conduit repository](https://github.com/ConduitIO/conduit), you can find [more examples](https://github.com/ConduitIO/conduit/tree/main/examples/pipelines), but to ilustrate a simple use case we'll show a pipeline using a file as a source, and another file as a destination. Check out the different [specifications](/docs/using/pipelines/configuration-file) to see the different configuration options.
+If you're using a macOS or Linux system, you can install Conduit with the
+following command:
 
+```shell
+$ curl https://conduit.io/install.sh | bash
+```
+
+If you're not using macOS or Linux system, you can still install Conduit
+following one of the different options provided
+in [our installation page](/docs/installing-and-running).
+
+:::note
+The Conduit binary contains both, the Conduit service and the Conduit CLI, with
+which you can interact with Conduit.
 :::
 
-Create a folder called `pipelines` at the same level as your Conduit binary.
-Inside of that folder create a file named `file-to-file.yml`.
+## Initialize Conduit 
+
+First, let's create a directory where we will build our first pipeline:
```

Comment:
I'd personally avoid this step. Let's assume people know where to create it. In the worst-case scenario, they delete the stuff where it's created, and then they'll do it again in the right place.

Reply:
I think you're right... I thought I'll need that path somewhere, but I didn't.